draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -559,6 +567,19 @@ defined:
   of the Verifiable Credential as described in (#schema-definition). `schema_uri` MUST NOT
   be used if `schema` is present.
 
+An example of a Type Metadata document is shown in (#ExampleTypeMetadata).
+
+## Extending Type Metadata {#extending-type-metadata}
+
+A type can extend another type. The extended type is identified by the URI in
+the `extends` property. Consumers MUST retrieve and process
+Type Metadata for the extended type before processing the Type Metadata for the extending
+type.
+
+The extended type MAY itself extend another type. This can be used to create a
+chain or hierarchy of types. The security considerations described in
+(#circular-extends) apply in order to avoid problems with circular dependencies.
+
```

Comment:
Why is this part of this PR?

Reply:
> Why is this part of this PR?

good catch and excellent question 

if I have my links right, this is already at 
https://github.com/oauth-wg/oauth-sd-jwt-vc/pull/236/files#diff-e750c0958ca10e395b4d3b77bb5aafccc668b4a1a50675dda79a8d90b39a2468R643